integrations/ultralytics-yolov3/recipes/yolov3-spp.transfer_learn_pruned.md

```diff
@@ -0,0 +1,158 @@
+      - model.3.conv.weight
+      - model.4.0.cv1.conv.weight
+      - model.4.0.cv2.conv.weight
+      - model.4.1.cv1.conv.weight
+      - model.4.1.cv2.conv.weight
+      - model.5.conv.weight
+      - model.6.0.cv1.conv.weight
+      - model.6.0.cv2.conv.weight
+      - model.6.1.cv1.conv.weight
+      - model.6.1.cv2.conv.weight
+      - model.6.2.cv1.conv.weight
+      - model.6.2.cv2.conv.weight
+      - model.6.3.cv1.conv.weight
+      - model.6.3.cv2.conv.weight
+      - model.6.4.cv1.conv.weight
+      - model.6.4.cv2.conv.weight
+      - model.6.5.cv1.conv.weight
+      - model.6.5.cv2.conv.weight
+      - model.6.6.cv1.conv.weight
+      - model.6.6.cv2.conv.weight
+      - model.6.7.cv1.conv.weight
+      - model.6.7.cv2.conv.weight
+      - model.7.conv.weight
+      - model.8.0.cv1.conv.weight
+      - model.8.0.cv2.conv.weight
+      - model.8.1.cv1.conv.weight
+      - model.8.1.cv2.conv.weight
+      - model.8.2.cv1.conv.weight
+      - model.8.2.cv2.conv.weight
+      - model.8.3.cv1.conv.weight
+      - model.8.3.cv2.conv.weight
+      - model.8.4.cv1.conv.weight
+      - model.8.4.cv2.conv.weight
+      - model.8.5.cv1.conv.weight
+      - model.8.5.cv2.conv.weight
+      - model.8.6.cv1.conv.weight
+      - model.8.6.cv2.conv.weight
+      - model.8.7.cv1.conv.weight
+      - model.8.7.cv2.conv.weight
+      - model.9.conv.weight
+      - model.10.0.cv1.conv.weight
+      - model.10.0.cv2.conv.weight
+      - model.10.1.cv1.conv.weight
+      - model.10.1.cv2.conv.weight
+      - model.10.2.cv1.conv.weight
+      - model.10.2.cv2.conv.weight
+      - model.10.3.cv1.conv.weight
+      - model.10.3.cv2.conv.weight
+      - model.11.cv1.conv.weight
+      - model.11.cv2.conv.weight
+      - model.12.cv1.conv.weight
+      - model.12.cv2.conv.weight
+      - model.13.conv.weight
+      - model.14.conv.weight
+      - model.15.conv.weight
+      - model.16.conv.weight
+      - model.19.cv1.conv.weight
+      - model.19.cv2.conv.weight
+      - model.20.cv1.conv.weight
+      - model.20.cv2.conv.weight
+      - model.21.conv.weight
+      - model.22.conv.weight
+      - model.23.conv.weight
+      - model.26.cv1.conv.weight
+      - model.26.cv2.conv.weight
+      - model.27.0.cv1.conv.weight
+      - model.27.0.cv2.conv.weight
+      - model.27.1.cv1.conv.weight
+      - model.27.1.cv2.conv.weight
+
+  - !GMPruningModifier
+    params:
+      - model.28.m.0.weight
+      - model.28.m.1.weight
+      - model.28.m.2.weight
+    init_sparsity: *init_sparsity
+    final_sparsity: *prune_low_target_sparsity
+    start_epoch: *pruning_start_epoch
+    end_epoch: *pruning_end_epoch
+    update_frequency: *pruning_update_frequency
+---
+
+# YOLOv3-SPP Pruned Transfer Learning
+
+This recipe transfer learns from a sparse, [YOLOv3-SPP](https://arxiv.org/abs/1804.02767) model.
+It was originally tested on the VOC dataset and achieved 0.84 mAP@0.5. 
+More information on the run can be found in the Weights and Biases project.
```

Comment:
add the link to the Weights and Biases project?

Reply:
good point, moved the statement to under the weights and biases section